lib/rest_5.js

```diff
@@ -484,20 +484,14 @@ function* resolveResult(result, chainId, node) {
     return (yield resolveResults([result], chainId, node))[0];
 }
 
-function* resolveResults(results, chainId, node) {
-    var count = 0;
-    var res = results;
-    while (count < 60 && res.filter(r => {return r.status === constants.PENDING}).length !== 0) {
-        res = yield getBlocResults(res.map(r => {return r.hash}), chainId, false, node);
-	yield promiseTimeout(1000);
-	count++;
-    }
-
-    if(count >= 60) {
-        throw new HttpError400('Transaction did not resolve');
+function* resolveResults(results) {
+    const pred = res => !res.some(r => r.status === constants.PENDING)
+    const act = node => getBlocResults(results.map(r => r.hash), false, node)
```

Comment:
Missing chainId parameter

Reply:
+1

Reply:
Also realizing the arguments were out of order in the old version too

Reply:
https://github.com/blockapps/blockapps-rest/pull/132/files